backend/src/preprocessing.py

```diff
@@ -151,50 +151,57 @@ def registration(packet):
     [in] packet : pipeline packet
     [post] images registered in place
     """
+    ALIGN = 1
+    REFERENCE = 0
```

Comment:
It might be better to pull this into constants.py to be consistent, but that's not a big deal for now. Final cleanup can come later